babel.config.js

```diff
@@ -61,7 +62,7 @@ const presets = [
   '@babel/preset-env',
   {
     'modules': isClosureCompiler ? false : 'commonjs',
-    'loose': true,
+    'loose': !isForUnitTesting,
```

Comment:
Actually, instead lets just add the class plugin with `loose: false` https://babeljs.io/docs/en/babel-plugin-transform-classes.

Reply:
This will apply to all classes, not just in tests, we are ok with that, right?

Reply:
I don't think dist uses this config, so that's fine.

Reply:
>  I don't think dist uses this config, so that's fine.

Running `gulp` locally I can see the class is using `_inherits` vs. `_inheritsLoose`, fyi.

Reply:
Yah, that's fine.